onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -567,5 +570,11 @@ void BackendManager::ShutdownBackendManager() {
   concrete_backend_.reset();
 }
 
+void BackendManager::RewindKVCache(size_t index) {
+  if (concrete_backend_) {
+    concrete_backend_->RewindKVCache(index);
+  }
+}
```

Comment:
What if concrete backend doesn't exist? should there be an exception

Reply:
Since GetOVCompiledModel() already handles nullptr by returning an empty model, it would be more consistent to make RewindKVCache a no-operation when the backend doesn't exist. Thus if concrete_backend_ is nullptr then it would do nothing in this case.